rootfs/usr/local/bin/startup

```diff
@@ -1,12 +1,25 @@
 #!/usr/bin/env sh
 
+# by https://unix.stackexchange.com/questions/146942/how-can-i-test-if-a-variable-is-empty-or-contains-only-spaces
+alias tes='test_empty_or_only_spaces'
```

Comment:
Can be better naming...
```suggestion
alias testEmptyVar='test_empty_or_only_spaces'
```

Or rename the function to be easier to read

Reply:
but the alias `tes` is wierd and not clear.

Reply:
Yes

`testEmptyVar` -> `test_empty_var` pls

Reply:
@Hydrog3n How now?